DataCollector/DoctrineDataCollector.php

```diff
@@ -26,9 +26,16 @@ class DoctrineDataCollector extends BaseCollector
     /** @var string[] */
     private $groupedQueries;
 
-    public function __construct(ManagerRegistry $registry)
+    /** @var bool */
+    private $shouldValidateSchemaClass;
```

Comment:
I would name it `shouldValidateSchema`. I don't see why there is `Class` in the name.

Reply:
done